### PR TITLE
[develop < T0093] Add query module arguments as tuples

### PR DESCRIPTION
**Pull request by Josipmrden**
### Description

Please briefly explain the changes you made here.

### Pull request type

- [x] Feature

### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

######################################

### Reviewer checklist (the reviewer checks this part)
- [ ] Core feature implementation
- [ ] Tests
- [ ] Code documentation
- [ ] Documentation on [memgraph/docs](https://github.com/memgraph/docs)

######################################

### CODE REVIEW

**Review by BorisTasevski — APPROVED** [review]

File: gqlalchemy/query_builder.py [hunk 1]
```diff
@@ -146,15 +146,21 @@ def construct_query(self) -> str:
...
         self.procedure = procedure
-        self.arguments = arguments
+        self.arguments = self._parse_query_arguments(arguments)
 
     def construct_query(self) -> str:
         return f" CALL {self.procedure}({self.arguments if self.arguments else ''}) "
 
+    def _parse_query_arguments(self, arguments: Optional[Union[str, Tuple[Union[str, int, float]]]]) -> str:
+        if isinstance(arguments, tuple):
+            return ", ".join([to_cypher_value(x) for x in arguments])
```
Comment (katarinasupe) [inline gqlalchemy/query_builder.py:160]: `x` is not descriptive, maybe use `arg` or `argument`.

File: gqlalchemy/query_builder.py [hunk 2]
```diff
@@ -146,15 +146,21 @@ def construct_query(self) -> str:
...
         super().__init__(DeclarativeBaseTypes.CALL)
 
         self.procedure = procedure
-        self.arguments = arguments
+        self.arguments = self._parse_query_arguments(arguments)
 
     def construct_query(self) -> str:
         return f" CALL {self.procedure}({self.arguments if self.arguments else ''}) "
 
+    def _parse_query_arguments(self, arguments: Optional[Union[str, Tuple[Union[str, int, float]]]]) -> str:
```
Comment (katarinasupe) [inline gqlalchemy/query_builder.py:158]: Similar methods were called `to_cypher_<>`, e.g., `to_cypher_qm_arguments()`, and located in [utilities](https://github.com/memgraph/gqlalchemy/blob/develop/gqlalchemy/utilities.py#L40). Let me know what you think, this is just a suggestion.

File: tests/docs/test_query_builder.py [hunk 3]
```diff
@@ -15,9 +15,20 @@
...
+from gqlalchemy.query_builder import CallPartialQuery
 from gqlalchemy.memgraph import Memgraph
 
 
+def test_call_procedure_arguments_string():
+    call_procedure = CallPartialQuery("dummy.procedure", "'a', 'b'").construct_query()
+    assert call_procedure == " CALL dummy.procedure('a', 'b') "
+
+
+def test_call_procedure_arguments_tuple():
```
Comment (katarinasupe) [inline tests/docs/test_query_builder.py:27]: Maybe mix up `int` and `str` in tuple just to check it all.

**Review by katarinasupe — CHANGES_REQUESTED** [review]

**Comment by brunos252** [comment]
I reviewed this PR, but won't approve so that it can't be merged withoud addressing @katarinasupe 's changes.

**Review by g-despot — APPROVED** [review]

**Comment by g-despot** [comment]
If there is no concrete reason for updating it, please refrain from pushing changes to the Poetry lock file. @Josipmrden

**Review by brunos252 — APPROVED** [review]

**Review by katarinasupe — APPROVED** [review]

**Pull request merged**